spec/controllers/groups_controller_spec.rb

```diff
@@ -326,5 +328,19 @@
         is_expected.to respond_with(:redirect)
       end
     end
+
+    describe 'POST #invite_member' do
+      before :each do
+        @grouping = create(:grouping_with_inviter, assignment: @assignment, inviter: @current_student)
```

Comment:
You aren't actually using the `@grouping` variable, so you can just call `create(...)` without assigning it to a variable. You can remove the `@grouping.reload` line below this one too.